src/qt/optionsdialog.cpp

```diff
@@ -94,20 +94,36 @@ OptionsDialog::OptionsDialog(QWidget *parent, bool enableWallet) :
 
     ui->lang->setToolTip(ui->lang->toolTip().arg(PACKAGE_NAME));
     ui->lang->addItem(QString("(") + tr("default") + QString(")"), QVariant(""));
+
+    /** Additional language names that aren't covered by QLocale as of version Qt 5.9.8*/
+    static const std::map<std::string, std::string> language_names = {
+        {"eo", "Esperanto"}, // TODO: Remove after Qt 5.12 - https://bugreports.qt.io/browse/QTBUG-57802?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&showAll=true
+        {"ku_IQ", "Kurdish - Iraq"},
+        {"la", "Latin"},
+        {"pam", "Kapampangan"},
+        {"szl", "Silesian"},
+    };
+
```

Comment:
Instead of commenting the code you could use `QT_VERSION` macro like here:

https://github.com/bitcoin/bitcoin/blob/dc5333d31f280e09bb1e8cdacfbe842f4ab9e69b/src/qt/guiutil.cpp#L885-L892

Reply:
thanks @hebasto added your suggestion

Reply:
@jonasschnelli @hebasto about the TODO we have this review. TODO or not TODO :)
Use `QT_VERSION_CHECK` and the comment seems good?